<ui version="4.0" >
 <class>remote</class>
 <widget class="QDialog" name="remote" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Push</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Local Branches</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="local_branch" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="local_branches" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Remote</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="remotename" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="remotes" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Destination Branch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="remote_branch" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="remote_branches" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="ffwd_only_checkbox" >
       <property name="text" >
        <string>Fast Forward Only</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="tags_checkbox" >
       <property name="text" >
        <string>Include tags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="rebase_checkbox" >
       <property name="text" >
        <string>Rebase</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="action_button" >
       <property name="text" >
        <string>Push</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>released()</signal>
   <receiver>remote</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>502</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel" >
     <x>274</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
